### ISSUE TITLE
Session opening alert did not go out to some courses/students

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#9197]
v6.13.0
Reported by a user.
The following two sessions have identical opening times. One session sent session-opening emails to the instructor but not the students; the other didn't send any emails (the user got suspicious because she received email from one session but not the other). I confirmed by checking logs. When I resent the emails using 'Remind' option, it worked for both sessions.

![image](https://user-images.githubusercontent.com/1673303/46991021-99963f80-d136-11e8-8382-a101c3072e95.png)

**Comment by xpdavid** [comment]
Do they have `sentOpenEmail == true` in the datastore?

**Comment by damithc** [comment]
Yes, but one of the courses has a deleted session in the recycle bin that has the exact same opening time. Maybe that has some relevance?

**Comment by damithc** [comment]
Both sessions have questions, students, and is visible from opening time. But it is possible that she enrolled students after the session was open. I'll check with her. Still doesn't explain why she received the instructor copy for one session but not the other while no emails were sent to students in both sessions.
